packages/clerk-js/src/utils/passkeys.ts

```diff
@@ -114,6 +142,7 @@ async function webAuthnGetCredential({
     const credential = (await navigator.credentials.get({
       publicKey: publicKeyOptions,
       mediation: conditionalUI ? 'conditional' : 'optional',
+      signal: __internal_WebAuthnAbortService.createAbortSignal(),
```

Comment:
Only one call to retrieve credentials can exist at time.

That means in the case of `autofill` user may never interact with it, this makes sure that every new call to retrieve the credentials will cancel the previews one.